cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh

```diff
@@ -28,14 +28,12 @@ function deploy_sriov_operator {
   fi
 
   pushd $operator_path
-    # TODO: right now in CI we need to use upstream sriov cni in order to have this
-    # https://github.com/intel/sriov-cni/pull/88 available. This can be removed once the feature will
-    # be merged in openshift sriov operator. We need latest since that feature was not tagged yet
-    sed -i '/SRIOV_CNI_IMAGE/!b;n;c\              value: nfvpe\/sriov-cni' ./deploy/operator.yaml
-    sed -i 's#image: quay.io/openshift/origin-sriov-network-operator$#image: quay.io/openshift/origin-sriov-network-operator:4.2#' ./deploy/operator.yaml
-    sed -i 's#value: quay.io/openshift/origin-sriov-network-config-daemon$#value: quay.io/openshift/origin-sriov-network-config-daemon:4.2#' ./deploy/operator.yaml
+    export RELEASE_VERSION=4.4
+    export SRIOV_NETWORK_OPERATOR_IMAGE=quay.io/openshift/origin-sriov-network-operator:${RELEASE_VERSION}
+    export SRIOV_NETWORK_CONFIG_DAEMON_IMAGE=quay.io/openshift/origin-sriov-network-config-daemon:${RELEASE_VERSION}
+    export OPERATOR_EXEC=${KUBECTL}
```

Comment:
https://github.com/kubevirt/kubevirtci/pull/134#discussion_r317153351

Reply:
Now we have multiple environment variables to export and I find this more readable than inline. And I want to keep it consistent. (Not sure I can export SHELL too, can I?)

Reply:
> (Not sure I can export SHELL too, can I?)

Yes, please

Reply:
Done